templates/user/unsubscribe_email__confirm.html

```diff
@@ -5,12 +5,11 @@
 
     {% if success %}
         <p>
-            Successfully Unsubscribed
+            Successfully unsubscribed
         </p>
     {% else %}
         <p>
-            Unsubscribe link is invalid!
-            you must be already unsubscribed
+            Unsubscribe link is invalid!  you must be already unsubscribed
```

Comment:
@eoglethorpe Can you provide the text for when the user clicks on the unsubscribe link after he has already been unsubscribed?